src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.cs

```diff
@@ -169,7 +169,7 @@ protected virtual NodeOperations CreateNodeOperationTasks(CancellationToken canc
             {
                 using (Logger.LogBlock(FunctionId.Formatting_CollectIndentBlock, cancellationToken))
                 {
-                    return AddOperations<IndentBlockOperation>(task.Result, (l, n) => _formattingRules.AddIndentBlockOperations(l, n, _token2), cancellationToken);
```

Comment:
💭 It's unclear to me what `_token2` means, and why it's still necessary.

Reply:
token1 and token2 is start and end token of the span given.